src/app/app.component.scss

```diff
@@ -1,3 +1,2 @@
 body{
-    background-color: blue;

```

Comment:
can u plz undo this, it cause a confliict

Reply:
Will edit it